src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,5 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android" package="com.amplitude">
+
+    <uses-permission android:name="android.permission.INTERNET" />
+
+</manifest>
```

Comment:
new line?

Reply:
Usually there's no need to add a new line at the end of the file.